common/djangoapps/third_party_auth/settings.py

```diff
@@ -100,7 +100,7 @@ def apply_settings(django_settings):
     django_settings.SOCIAL_AUTH_INACTIVE_USER_URL = '/auth/inactive'
 
     # Context processors required under Django.
-    django_settings.SOCIAL_AUTH_UUID_LENGTH = 4
+    django_settings.SOCIAL_AUTH_UUID_LENGTH = 16
```

Comment:
Suggest going with a lower value than 16 so we don't end up with usernames like `Matt_a1b2c3d4e5f6g7h8`

Reply:
Let me try this at 16 and we can tweak.